packages/kestrel_core/src/kestrel/mapping/data_model.py

```diff
@@ -298,18 +304,35 @@ def translate_projection_to_native(
 
 
 @typechecked
-def translate_projection_to_ocsf(
-    dmm: dict,
+def translate_projection_entity_to_ocsf(
+    to_ocsf_flattened_field_map: dict, native_projection: str
+) -> str:
+    _map = to_ocsf_flattened_field_map
+    if not native_projection.endswith("*"):
+        native_projection += ".*"
+    ocsf_projection = _map.get(native_projection, native_projection)
+    if isinstance(ocsf_projection, list):
+        ocsf_projection = ocsf_projection[0]
+    ocsf_projection = ocsf_projection[:-2]
+    return ocsf_projection
+
+
+@typechecked
+def translate_projection_attrs_to_ocsf(
```

Comment:
Same as above; seems like it should be `translate_attr_projection_to_ocsf`

Reply:
Good point!